include/z64bgcheck.h

```diff
@@ -198,6 +261,43 @@ typedef struct {
     /* 0x20 */ s32 chkOneFace; // bccFlags & 0x8
     /* 0x24 */ f32* distSq;    // distance from posA to poly squared
     /* 0x28 */ f32 chkDist;    // distance from poly
-} DynaLineTest;
+    /* 0x2C */ Actor* actor;
+    /* 0x30 */ s32 bgId;
+} DynaLineTest; // size = 0x34
+
+typedef struct {
+    /* 0x00 */ CollisionPoly* poly;
+    /* 0x04 */ Vec3s* vtxList;
+    /* 0x08 */ Vec3f* posA;
+    /* 0x0C */ Vec3f* posB;
+    /* 0x10 */ Vec3f* planeIntersect;
+    /* 0x14 */ s32 chkOneFace;
+    /* 0x18 */ f32 chkDist;
```

Comment:
```suggestion
    /* 0x14 */ s32 checkOneFace;
    /* 0x18 */ f32 checkDist;
```
This is just personal preference to write out the full word, it's only two extra letters. `chk` is used a lot in these bgcheck structs so I would prefer if they all changed, but you could also disregard this if it's not to your preference

Reply:
sure. I also changed this for the other `chk` variables in the file.